content/tracing/setup/go.md

```diff
@@ -80,53 +74,94 @@ ___
+
+```go
+package main
 
-Datadog APM client that implements an [OpenTracing][3] Tracer.
+import "gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
 
-**Initialization**
+func main() {
+    // Start the tracer with zero or more options.
+    tracer.Start(tracer.WithServiceName("my-service"))
+    defer tracer.Stop()
 
-To start using the Datadog Tracer with the OpenTracing API, you should first initialize the tracer with a proper `Configuration` object:
+    // Create a span for a web request at the /posts URL.
+    span := tracer.StartSpan("web.request", tracer.ResourceName("/posts"))
+    defer span.Finish()
+
+    // Set metadata
+    span.SetTag("my_tag", "my_value")
+}
+```
+
+## OpenTracing Support
+
+Import the [`opentracer` package][opentracing godoc] to expose the Datadog tracer as an [OpenTracing][3] compatible tracer.
+
+### Example
+
+A basic usage would be:
 
 ```go
+package main
+
 import (
-  // ddtrace namespace is suggested
-  ddtrace "github.com/DataDog/dd-trace-go/opentracing"
-  opentracing "github.com/opentracing/opentracing-go"
+    "github.com/opentracing/opentracing-go"
+
+    "gopkg.in/DataDog/dd-trace-go.v1/ddtrace/opentracer"
+    "gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
 )
 
 func main() {
-  // create a Tracer configuration
-  config := ddtrace.NewConfiguration()
-  config.ServiceName = "api-intake"
-  config.AgentHostname = "ddagent.consul.local"
-
-  // initialize a Tracer and ensure a graceful shutdown
-  // using the `closer.Close()`
-  tracer, closer, err := ddtrace.NewTracer(config)
-  if err != nil {
-    // handle the configuration error
-  }
-  defer closer.Close()
-
-  // set the Datadog tracer as a GlobalTracer
-  opentracing.SetGlobalTracer(tracer)
-  startWebServer()
+    // Start the regular tracer and return it as an opentracing.Tracer interface. You
+    // may use the same set of options as you normally would with the Datadog tracer.
+    t := opentracer.Start(tracer.WithServiceName("my-service"))
+
+    // Stop it using the regular Stop call.
+    defer tracer.Stop()
+    
+    // Set the global OpenTracing tracer.
+    opentracing.SetGlobalTracer(t)
+
+    // Use the OpenTracing API as usual.
 }
 ```
 
-Function `NewTracer(config)` returns an `io.Closer` instance that can be used to gracefully shutdown the `tracer`. It's recommended to always call the `closer.Close(), otherwise internal buffers are not flushed and you may lose some traces.
+**Note**: Using the [OpenTracing API][4] in parallel with the regular API or our integrations is fully supported. Under the hood, all of them
+make use of the same tracer. Make sure to check out the [API documentation][opentracing godoc] for more examples and details.
+
+## Sampling / Distributed Tracing
 
-**Usage**
+See your end-to-end performances by propagating a single trace across multiple services with distributed tracing. For more details about how to use and configure distributed tracing, check out the [package documentation][tracer godoc].
```

Comment:
Propagate a single trace across multiple services with distributed tracing. For more details about how to use and configure distributed tracing, check out the [godoc page][tracer godoc].